aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -324,6 +324,8 @@ When deploying apps to servers with [Web Deploy](/iis/publish/using-web-deploy/i
 
     ASP.NET Core runs in a separate process and manages the runtime. ASP.NET Core doesn't rely on loading the desktop CLR. Setting the **.NET CLR version** to **No Managed Code** is optional.
 
+1. For 64-bit application make sure that Enable 32-Bit Applications is set to false either under the **Set Application Pool Defaults** or under **Advanced Settings** for your specific application pool.
```

Comment:
Clarify this only affects standalone deployment. If you publish for a `Framework Dependent` app, this isn't affected.